.github/workflows/test-admin-deploy.yaml

```
@@ -112,8 +110,7 @@ jobs:
               --principal "*" \
               --function-url-auth-type NONE > /dev/null 2>&1
 
-            URL=$(aws lambda get-function-url-config --function-name $FUNCTION_NAME-$PR_NUMBER --query 'FunctionUrl' --output text)
-            echo "URL=$URL" >> $GITHUB_ENV
+            URL="$(aws lambda create-function-url-config --function-name $FUNCTION_NAME-$PR_NUMBER --auth-type NONE | jq .FunctionUrl)"
```

Since you'll capture this URL again lower down, what if you just created the URL config at this point like so?
```suggestion
            aws lambda create-function-url-config --function-name $FUNCTION_NAME-$PR_NUMBER --auth-type NONE | jq .FunctionUrl
```
And then you could remove the `echo "pr_url=$URL" >> $GITHUB_OUTPUT` just below as well.

---

Hmm good point.  Now that I look more closely I didn't mean to get rid of the part that sets `URL` though - since that is what's used to have the bot comment on the PR.  I'll fix up both.

---

Ah I had meant to remove line 117 but keep line 116, so I'll need to continue setting `URL` so that the workflow only posts the PR review env comment once on creation.